recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -0,0 +1,23 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+LICENSE = "CLOSED"
+
+UBOOT_CONFIG[doma] = "xenguest_arm64_defconfig"
+UBOOT_CONFIG_prepend = "doma "
+
+SRCREV = "${AUTOREV}"
+SRC_URI = "\
+    git://github.com/arminn/u-boot.git;protocol=https;branch=mr_rcar_pvblock; \
+"
+
+GUEST = "${@bb.utils.contains('XT_GUESTS_INSTALL', 'doma', 'doma', 'domu', d)}"
+
+FILES_${PN} += " \
+   /xt/${GUEST} \
```

Comment:
Do not use "/xt" we have XT_DIR_ABS_ROOTFS for that which may change

Reply:
fixed